lms/envs/common.py

```diff
@@ -1852,6 +1852,11 @@
 # parallel, and what the SES rate is.
 BULK_EMAIL_RETRY_DELAY_BETWEEN_SENDS = 0.02
 
+############################# Persistent Grades ####################################
+
+# Queue to use for updating persistent grades
+RECALCULATE_GRADES_ROUTING_KEY = LOW_PRIORITY_QUEUE
```

Comment:
If you add the line in aws.py, it gives us the flexibility to make an edX only queue for our volume and configure it in ansible while leaving most deployments with the low priority queue.